# Status: submitted to linux-serial

Subject: [PATCH] serial: Fix compilation error in dz.c

Fix the following compilation error in the dz serial driver that got
introduced with the "kernel console should send CRLF not LFCR" change.

  CC      drivers/serial/dz.o
drivers/serial/dz.c: In function 'dz_console_putchar':
drivers/serial/dz.c:679: error: 'uport' undeclared (first use in this function)
drivers/serial/dz.c:679: error: (Each undeclared identifier is reported only once
drivers/serial/dz.c:679: error: for each function it appears in.)

Signed-off-by: Martin Michlmayr <tbm@cyrius.com>


--- a/drivers/serial/dz.c
+++ b/drivers/serial/dz.c
@@ -674,7 +674,7 @@ static void dz_reset(struct dz_port *dpo
 }
 
 #ifdef CONFIG_SERIAL_DZ_CONSOLE
-static void dz_console_putchar(struct uart_port *port, int ch)
+static void dz_console_putchar(struct uart_port *uport, int ch)
 {
 	struct dz_port *dport = (struct dz_port *)uport;
 	unsigned long flags;

